VikingQuiz.UI/src/Components/MyQuizesPage/ManageQuizComponent/ManageQuizComponent.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import './ManageQuizComponent.css';
+import {Link, Redirect} from 'react-router-dom';
+import axios from '../../../../node_modules/axios';
+import YesNoComponent from '../../YesNoComponent/YesNoComponent';
+
+class ManageQuizComponent extends React.Component<any, any> {
+    constructor(props: any) {
+      super(props);
+
+      this.state = {
+          serverMessage: '',
+          redirect: false,
+          isHidden: true,
+          id: this.props.id
+      }
+
+      this.handleDeleteButtonClick = this.handleDeleteButtonClick.bind(this);
+      this.handleQuizDeleteNoClick = this.handleQuizDeleteNoClick.bind(this);
+      this.handleQuizDeleteYesClick = this.handleQuizDeleteYesClick.bind(this);
+      this.handleEditQuizRedirect = this.handleEditQuizRedirect.bind(this);
+    }
+
+
+    public componentDidMount(){
+        this.setState( {id: this.props.id})   
+    }
+
+
+    public handleDeleteButtonClick(){
+        this.setState({isHidden: false})
+        this.props.deleteSelection();
+    }
+
+
+    public handleQuizDeleteNoClick(props: any){
+        this.setState({isHidden: true})
+        this.props.noDeleteAnswer()
+    }
+
+
+    public handleQuizDeleteYesClick(props: any){
+        
+        const url : string = 'http://localhost:60151/api/quizzes/' + this.props.id
```

Comment:
Extract this url as a constant at the beginning of the class. Don't forget to give it a meaningful name like; url is too general, what about "apiAddress". Since you can't declare it as a constant there, try "private readonly"